### ISSUE TITLE
Parsing errors which may affect assistive technology

### ISSUE THREAD

**Issue by dougmacknz** [cultureamp/kaizen-design-system#2083]
TLDR: we need to replace any `<div>`s living inside of a `<label>` with `<span>`s, being careful about styling.

#### Parsing errors which may affect assistive technology
Type:WCAG 2
Principle: Robust
Guideline:4.1 Compatible
Success Criteria: 4.1.1 Parsing (A)
Level: A
Platform: Web
Severity:Medium

#### What's the issue?
The survey pages contains parsing errors.
This includes:
incorrect nesting of elements - <div> cannot be child of <label>

#### Why it matters
User agents, including assistive technologies, require code to be semantically correct so they can accurately interpret and parse content. For example, a screen reader may say the wrong label if a form uses the same ID to identify multiple fields, causing the user to make mistakes.

#### Our recommendation
To check for parsing errors, we use the Nu HTML Checker and the suggested bookmarklets.
* Go to About the Nu HTML Checker.
* From there, add the following bookmarklets to Chrome.     
* Check serialized DOM of the current page
* Check for WCAG 2.0 parsing compliance
* Run the 'Check serialized DOM of current page 'bookmarklet on the page you are testing. It will load the DOM into Nu HTML Checker and display results.
* Then run the 'Check for WCAG 2.0 parsing compliance'. This will filter the results to only display parsing errors.
* Check that there are no parsing errors:     
* Missing start or end tags
* Improper nesting of elements
* Duplicate attributes

Fix the HTML errors reported by the Nu HTML Checker to reduce the risk of parsing issues impacting on assistive technology users.

**Comment by ActuallyACat** [comment]
@aaronknol This is related to the issue we were talking about with Box. The tl;dr is we should add some kind of `as` or `tag` prop to `Box` so it can render elements other than divs.
